cypress/component/QueryBuilder/QueryBuilder.spec.tsx

```diff
@@ -260,4 +271,42 @@ describe('QueryBuilder', () => {
       getGroupByDepth(4).should('be.visible').and('exist')
     })
   })
+
+  describe('adds, clones and removes rules', () => {
```

Comment:
to keep description consistent with groups tests 
```suggestion
  describe('we manipulate query builder rules', () => {
```

Reply:
will do.